utils/form_validation.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+/** Sanitize the input of a form. Preferable to do on all fields because of form/request forging */
+function clean($data)
+{
+    $data = trim($data);
```

Comment:
I have a deprecation warning when `$data == null`. It happens when I open the page "create an event" => I think the error occurs at form init.

Reply:
Mmm that's weird. I guess I could add a null check then 🙂

Reply:
Done!